Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -9709,67 +9709,67 @@
       "localizations" : {
         "de" : {
           "stringUnit" : {
-            "state" : "translated",
+            "state" : "needs_review",
             "value" : "wss://yourrelay.com"
```

Comment:
I wonder if we just remove the `wss://` from all of these. Seems simple enough to me, even though I don't speak most of these languages.

Reply:
I thought I edited those values to remove the wss:// part. I will look at this again.